dpnp/dpnp_array.py

```diff
@@ -1317,21 +1317,31 @@ def transpose(self, *axes):
 
         For full documentation refer to :obj:`numpy.ndarray.transpose`.
 
+        Parameters
+        ----------
+        axes : None, tuple or list of ints, n ints, optional
+            * ``None`` or no argument: reverses the order of the axes.
+            * tuple or list of ints: `i` in the `j`-th place in the tuple/list
+            means that the array’s `i`-th axis becomes the transposed
+            array’s `j`-th axis.
+            * n ints: same as an n-tuple/n-list of the same ints (this form is
```

Comment:
in the generated documentation `n ints` is not shown as an element of the list due to some reason